Sources/SKTestSupport/Utils.swift

```diff
@@ -51,7 +51,6 @@ public func testScratchDir(testName: String = #function) throws -> URL {
   let url = FileManager.default.temporaryDirectory
     .realpath
     .appendingPathComponent("sourcekit-lsp-test-scratch")
-    .appendingPathComponent(UUID().uuidString)
```

Comment:
If we're not adding the UUID, it might be worth deleting the folder if it exists (probably in each workspace?)

Reply:
Oh, good catch. I forgot to add that.